include/wx/textctrl.h

```diff
@@ -509,6 +526,8 @@ class WXDLLIMPEXP_CORE wxTextAttr
     wxFontWeight        m_fontWeight;
     wxFontFamily        m_fontFamily;
     bool                m_fontUnderlined;
+    wxTextAttrUnderlineType m_fontUnderlinedType;
+    wxColour            m_fontUnderlineColor;
```

Comment:
Please call it `m_colUnderline` for consistency with the other colours here.

Reply:
Will be fixed in the upcoming commit.